staging/src/k8s.io/apiextensions-apiserver/pkg/apis/testdata/HEAD/apiextensions.k8s.io.v1beta1.CustomResourceDefinition.yaml

```diff
@@ -212,6 +214,8 @@ spec:
           optionalOldSelf: true
           reason: reasonValue
           rule: ruleValue
+    selectableFields:
```

Comment:
> note to self to check serialization because once I forgot to ensure that when not set, no different serialization was created.

modified HEAD, but not one of the older ones.